### ISSUE TITLE
CPU Temp not displayed

### ISSUE THREAD

**Issue by TobiasKneidl** [afaqurk/linux-dash#451]
My CPU Temperatur is not displayed.

Output of Linux-Dash: 
Temprature | /usr/bin/sensors SA °C

Output of sensors:

coretemp-isa-0000
Adapter: ISA adapter
Core 0:       +28.0°C  (high = +105.0°C, crit = +105.0°C)
Core 1:       +28.0°C  (high = +105.0°C, crit = +105.0°C)
Core 2:       +26.0°C  (high = +105.0°C, crit = +105.0°C)
Core 3:       +26.0°C  (high = +105.0°C, crit = +105.0°C)

Output sensors -v:

sensors version 3.4.0 with libsensors version 3.4.0

**Comment by zmrow** [comment]
Is this on a raspberry pi?

**Comment by TobiasKneidl** [comment]
No. It's an intel chip. 

Model name | Intel(R) Celeron(R) CPU J1900 @ 1.99GHz
-- | --

**Comment by VIXNESS** [comment]
yes.This problem happens on raspberry pi 3 debian 9 sometime.

**Comment by TobiasKneidl** [comment]
Yes, but it's an Intel(R) Celeron(R) CPU J1900 @ 1.99GHz with Ubuntu 17.10 4.13.0-37-generic

**Comment by Jiab77** [comment]
To get the main temperature data, just change `raspbian` by `ubuntu` in the `cpu_temp()` function in the file `app/server/linux_json_api.sh`.

If you don't have an `ubuntu` based distrib, check the value of `ID` in file `/etc/os-release` and set it in place of `raspbian` in the script.

For temperature data by cores, see this pull request https://github.com/afaqurk/linux-dash/pull/465 (not tested yet)
